lib/perl/Genome/Model/Tools/Sx/Trim/Flexbar.pm

```diff
@@ -281,6 +282,18 @@ sub _resolve_adapters {
     return 1;
 }
 
+sub _resolve_length_dist_parameter {
+    my $self = shift;
+
+    my $no_length_dist = $self->no_length_dist;
+
+    if ( $self->version >= 230 ) {
```

Comment:
The other special cases are checking `$self->version eq 230`.  I would put a small vote towards consistency and keeping this method next to the others that are also handling special cases for 230.